.gitignore

```diff
@@ -129,3 +129,6 @@ dmypy.json
 .pyre/
 
 .vim
+
+# We hate Macs :smiling_face_with_3_hearts:
```

Comment:
Nit: Let's not leave this comment here

Reply:
I think for community projects like this it's good if there's some "flair" in the source code that makes it not-boring to read / parse / skim through, but I'll remove it